### PR TITLE
decomp some small files, do some cleanup

### PR DESCRIPTION
**Pull request by barzamin**
this pr is a bit of a random combination of various things i've been working on/cleaning up idly by looking at the smallest remaining `.s` files

### CODE REVIEW

File: src/d/d_save/d_save_init.cpp [hunk 1]
```diff
@@ -1 +1,5 @@
-// ok
+#include "d/d_save/d_save_init/d_save_init.h"
+
+void setInitEventBit() {
+    return;
+}
```
Comment (notyourav) [inline src/d/d_save/d_save_init.cpp:5]: FYI, ending on anything but a newline violates C standard. A lot of text editors will add them back anyway.

**Review by notyourav — APPROVED** [review]

File: src/d/d_save/d_save_init.cpp (hunk 1 above)
Comment (barzamin) [reply src/d/d_save/d_save_init.cpp:5]: i am not sure why my editor didn't add a final \n. i feel like clang-format should've regardless! anyway, i can fix it by hand if it's an issue

Comment (notyourav) [reply src/d/d_save/d_save_init.cpp:5]: No need, just wanted to mention in case you were trying to delete them manually.

**Comment by barzamin** [comment]
@Pheenoh fixed merge, sorry for the misc PR with lots of conflict points 😅

**Pull request merged**